drivers/clock_control/nrf_power_clock.c

```diff
@@ -30,6 +30,14 @@ LOG_MODULE_REGISTER(clock_control, CONFIG_CLOCK_CONTROL_LOG_LEVEL);
 #define INF(dev, subsys, ...) CLOCK_LOG(INF, dev, subsys, __VA_ARGS__)
 #define DBG(dev, subsys, ...) CLOCK_LOG(DBG, dev, subsys, __VA_ARGS__)
 
+/* If LF RC is started within given window after soft reset occurred then
+ * LF RC fails to start. A workaround is applied that time from reset is
+ * measured (using systick) and attempt to start LF clock is delayed if it would
+ * occur in that window.
+ */
+#define NRF52_ANOMALY_132_WINDOW_START_US 225
+#define NRF52_ANOMALY_132_WINDOW_END_US 330
```

Comment:
Values taken from test. window `[66.138]` is related to the moment when clock got stopped reported only by `LFCLKSTAT` and there is no interrupt for that so it is quite impractical to use since you could only relate to that by polling `LFCLKSTAT`. I've reported it to anomaly authors to update that.
When soft reset ccurs lfclk is implicitly stopped so same window applies.